### PR TITLE
Include notice and license in published artifacts, switch them to md …

### PR DESCRIPTION
**Pull request by lukasj**
…format

Signed-off-by: Lukas Jungmann <lukas.jungmann@oracle.com>

### CODE REVIEW

File: NOTICE.md [hunk 1]
```diff
@@ -82,6 +84,14 @@ Apache Ant (1.7.0)
 
 * License: Apache License, 2.0
 
+Apache Ant (1.10.7)
+
+* License: Apache-2.0 AND W3C AND LicenseRef-Public-Domain
+
+Apache Ant (1.10.7)
```
Comment (rfelcman) [inline NOTICE.md:91]: It's twice there.

File: NOTICE.md [hunk 2]
```diff
@@ -125,6 +135,29 @@ ASM (6.1.1)
...
+
+ASM (7.3.1)
+
+* License: BSD-3-Clause
+
+ASM (8.0.1)
+
+* License: BSD-3-Clause
+
+ASM 9 (n/a)
```
Comment (rfelcman) [inline NOTICE.md:158]: ASM  (9.0)

License: BSD-3-Clause

File: NOTICE.md [hunk 3]
```diff
@@ -494,13 +604,65 @@ SLF4J API (1.7.2)
...
+* License: MIT
+
 spring-agent (2.5.3)
 
 * License: Apache License, 2.0
 * Project: http://www.springframework.org
 * Source:
    http://downloads.sourceforge.net/springframework/spring-framework-2.5.3-with-dependencies.zip?modtime=1204283106&big_mirror=0
 
+spring-aop (5.2.0)
```
Comment (rfelcman) [inline NOTICE.md:622]: I think, that all spring components are under Apache-2.0 license.

**Review by rfelcman — COMMENTED** [review]
_about.html, ..._ movement to _features_ directory. Are these html files used now?

File: NOTICE.md (hunk 1 above)
Comment (lukasj) [reply NOTICE.md:91]: it is generated file

File: NOTICE.md (hunk 2 above)
Comment (lukasj) [reply NOTICE.md:158]: again, generated file

File: NOTICE.md (hunk 3 above)
Comment (lukasj) [reply NOTICE.md:622]: do you see Apache-2.0 license missing somewhere? Some pieces are dual-licensed

**Comment by lukasj** [comment]
> _about.html, ..._ movement to _features_ directory. Are these html files used now?

do we build _features_? in either case, html files make sense for eclipse-plugin packaging type or _features_. It makes no sense to have them in plain jars

**Review by rfelcman — APPROVED** [review]
LGTM

**Pull request merged**